static_src/util/cf_api.js

```diff
@@ -115,8 +115,8 @@ export default {
       const reqs = urls.map((u) => http.get(u).then((r) => r.data.resources));
 
       return Promise.all(reqs)
-        .then((all) => all.pop())
-        .then((all) => [].concat.call([], res.data.resources, all))
+        .then((all) => [].concat.apply([], all))
```

Comment:
This can just be `[].concat(res.data.resources, all)`, otherwise you're creating two empty arrays and throwing one away, right?